#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_add_search_engines.dpatch by Jose Carlos Medeiros <debian@psabs.com.br>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add more SearchEngines in .conf file

#################################
# SearchEngine field for the sensis.com.au search engine
# http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=327496
# From: Gavin Rogers <grogers@vk6hgr.echidna.id.au>

################################
# Support for French versions of Google keywords
# http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=253830
# From: Cyril Bouthors <cyb@debian.org>


@DPATCH@
Index: webalizer-2.01.10-vc/sample.conf
===================================================================
--- webalizer-2.01.10-vc.orig/sample.conf	2006-10-27 14:31:29.000000000 +0400
+++ webalizer-2.01.10-vc/sample.conf	2006-10-27 14:31:34.000000000 +0400
@@ -530,6 +530,12 @@
 SearchEngine	mamma.com	query=
 SearchEngine	alltheweb.com	query=
 SearchEngine	northernlight.com  qr=
+SearchEngine	sensis.com.au	find=
+SearchEngine	google.nl	q=
+SearchEngine	google.fr	q=
+SearchEngine	google.ch	q=
+SearchEngine	google.ca	q=
+SearchEngine	google.be	q=

 # The Dump* keywords allow the dumping of Sites, URL's, Referrers
 # User Agents, Usernames and Search strings to seperate tab delimited
